app/views/ui-components/v1/forms/broker_registration/_personal_information.html.slim

```diff
@@ -32,8 +32,13 @@ fieldset
 
     .col-md-4.mb-2
         label for="inputNPN"  NPN *
-
-        = f.text_field :npn, placeholder:'NPN', required: !EnrollRegistry.feature_enabled?(:allow_alphanumeric_npn), id:'inputNPN', class:'form-control', maxlength:'10', onkeypress: EnrollRegistry.feature_enabled?(:allow_alphanumeric_npn) ? 'return isAlphaNumeric(event);' : 'return isNumberKey(event);'
+        = f.text_field :npn,
+          placeholder: 'NPN',
+          required: controller.action_name == 'new' ? 'true' : !EnrollRegistry.feature_enabled?(:allow_alphanumeric_npn),
```

Comment:
@hKalesnikavaideacrew could we do

`
required: controller.action_name == 'new' ? EnrollRegistry.feature_enabled?(:allow_alphanumeric_npn) : !EnrollRegistry.feature_enabled?(:allow_alphanumeric_npn)`